components/utils/data.js

```diff
@@ -0,0 +1,160 @@
+const data = [
+  {
+    userName: "Julian",
+    userPictureSrc: "",
+    userEmail: "Julian@web.de",
+    userPhonenumber: "+1235794144",
+    id: 1,
+    adPictureSrc: "",
+    adTitle: "Build an App",
+    adDescription: "I can be a Hero in building an App for you.",
+    adCosts: 99,
+    category: "Technique",
+    createdDate: "16.11.2022",
```

Comment:
```suggestion
    createdDate: new Date(2022,10,16),
```
If you want to work with dates, you need the `date` data type.
Side note: the month starts with 0 = January which is why 10 = November.

You can apply this to every object in this array.